code/modules/parallax/parallax_render_sources/parallax_render_source_groups.dm

```diff
@@ -106,6 +106,14 @@ Z_LEVEL_PARALLAX_RENDER_SOURCE_GROUP(5)
 		/atom/movable/screen/parallax_render_source/space_2,
 	)
 
+/datum/parallax_render_source_group/area/watchful_eye_sensor
+	parallax_render_source_types = list(
+		/atom/movable/screen/parallax_render_source/void,
```

Comment:
This isn't meant to be in the void so probably shouldn't use the void parallax?

Reply:
used the void parallax to try and indicate typhon's purple colours but good point

Reply:
changed it to space_2, but kept the void clouds